2025-04-30/agenda.md

```diff
@@ -0,0 +1,31 @@
+# Agenda for MDN Community meeting - 30th April, 2025
+
+**Join details:** See 'where' in [the calendar invite](https://calendar.google.com/calendar/u/0/embed?src=c_4656dd7c36825e2be115c0e7992191d550d16edcec37151eb6018581f654727b@group.calendar.google.com) for the call URL.
+
+## Intro & overview of agenda
+
+- Rules: use the chat for questions or to request to speak so that we're talking in-turns.
+
+## Team updates
+
+Project updates from the MDN team
+
+## Contributor updates
+
+Project/work updates from contributors
+
+- [example] Progress on my awesome project. See [link for more details](https://github.com/mdn/community-meetings) (@MyUsername)
+
+## Questions/Discussions
+
+Questions from folks & anything to discuss
+
+- [example] Discuss my interesting topic. See [link for more details](https://github.com/mdn/community-meetings) (@MyUsername)
+
+## Technical Writing Workshop
+
+Workshop by Pranit Brahmbhatt, Technical Content Developer at Dell.
+
+Questions for the Facilitator
+
+- [example] Can you tell me more about my interesting question? See [link for more details](https://github.com/mdn/community-meetings) (@MyUsername)
```

Comment:
I feel this format is not always clear to contributors. Also, in our own updates, we lead with our name, so we could mirror that format here.
```suggestion
- (@YourUsername) Can you tell me more about <add your question or topic>? See <[add your link](url)> for more details.
```

Reply:
Makes sense. Thank you!